Jenkinsfile-deploy

```diff
@@ -0,0 +1,64 @@
+def releaseBranch = "release-${env.STAGE}"
```

Comment:
This is also how I've done it in the frontend and API projects, but we should maybe rethink it when we have a staging environment. It might be useful to use the same name for the release branches as for the Docker tags, so that you can specify "intg" when you deploy to staging, rather than a specific version tag.

Reply:
OK